dts/bindings/ipc/subsys,ipc-openamp-static-vrings.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: OpenAMP (RPMsg with static VRINGs) backend
+
+compatible: "subsys,ipc-openamp-static-vrings"
+
+include: base.yaml
+
+properties:
+  role:
+    description: OpenAMP roles
+    required: true
+    type: string
+    enum:
+      - primary
+      - secondary
+
+  shm:
+    description: phandle to the shared memory region
+    required: true
+    type: phandle
+
+  mboxes:
+    description: phandle to the MBOX controller
```

Comment:
It looks the backend requires at least `tx` and `rx` mailboxes described. Should we add it to the description?

Reply:
Is possible to define twice the same mailbox? 
Regarding https://github.com/zephyrproject-rtos/zephyr/pull/38322
The same mailbox is used in RX and TX...

Reply:
> It looks the backend requires at least tx and rx mailboxes described. Should we add it to the description?

Yes, I'll add it.

> Is possible to define twice the same mailbox?

yes, you can use the same mailbox and you can even use the same channel for RX and TX if your hardware (and driver) supports it.